app/code/Magento/Customer/Test/Mftf/Section/AdminCustomerWishlistSection.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
+    <section name="AdminCustomerWishlistSection">
+        <element name="productName" type="input" selector="#wishlistGrid_filter_product_name"/>
+        <element name="searchButton" type="button" selector=".action-default.scalable.action-secondary"/>
+        <element name="deleteButton" type="text" selector=".even > td:nth-child(7) > a:nth-child(1)"/>
```

Comment:
This is very flaky way to find the element. 
If there is no other option, even selecting by label is more appropriate